Model/Method/Tinka.php

```diff
@@ -21,10 +21,11 @@
 
 namespace Buckaroo\Magento2\Model\Method;
 
-use Magento\Quote\Model\Quote\AddressFactory;
-use Magento\Catalog\Model\Product\Type;
-use Magento\Tax\Model\Calculation;
 use Magento\Tax\Model\Config;
```

Comment:
use Magento\Tax\Model\Config;
use Magento\Tax\Model\Calculation; 
can be removed